change-notes/1.20/analysis-python.md

```diff
@@ -16,6 +16,7 @@
 
  | **Query**                  | **Expected impact**    | **Change**                                                       |
 |----------------------------|------------------------|------------------------------------------------------------------|
+| Unused import (`py/unused-import`) | Fewer false positive results | Results where the imported module is used in a doctest string are no longer reported |
```

Comment:
Thanks for the change note. Please can we have "doctest" in backticks and a period at the end of the sentence.